auth-web/src/components/auth/create-account/UserProfileForm.vue

```diff
@@ -495,7 +495,7 @@ export default class UserProfileForm extends Mixins(NextPageMixin, Steppable) {
 
     private navigateBack (): void {
       if (this.currentOrganization) {
-        this.$router.push(`/account/${this.currentOrganization.id}`)
+        window.location.assign(configHelper.getBcrosDashboardURL())
```

Comment:
wouldn't this refresh the page?. Or is it expected to be that way :)

Reply:
It would yes, since we are navigating back to our Uber Dashboard (higher level page) and not a local route.
Is there an alternative and/or better approach when redirecting?